arbnode/node.go

```diff
@@ -779,7 +779,7 @@ func CreateNode(
 	}
 	if currentNode.StatelessBlockValidator != nil {
 		apis = append(apis, rpc.API{
-			Namespace: "arbvalidator",
+			Namespace: "arbdebug",
```

Comment:
Do we need to worry about existing clients of the API which might still be expecting it in the previous namespace?
(i.e. Should we expose the same method in both the old and new namespace for a transition period?)

Reply:
Totally see your point. From my understanding of the issue, I think we're breaking them deliberately since it wasn't supposed to be exposed in that namespace originally.

Reply:
We're deliberately moving it to stress the point that this is a debug functionality and not the way to validate blocks (because you must validate all blocks up to X to validate X).
If anyone is using this interface today they are probably wrong, and should use block_validator instead.